main/solution/post-deployment/serverless.yml

```diff
@@ -71,7 +71,7 @@ custom:
   registrationWAF:
     albExportName: "${self:custom.settings.awsRegionShortName}-${self:custom.settings.solutionName}-LoadBalancerFullName"
     name: "${self:custom.settings.awsRegionShortName}-${self:custom.settings.solutionName}-WAF"
-    rateLimit: 1000 # Per ip, per 5 minutes
+    rateLimit: ${self:custom.settings.wafIpRateLimit} # Per ip, per 5 minutes
```

Comment:
Might be good to move this comment `# Per ip, per 5 minutes` to the defaults file where it's more likely to be found.

Reply:
sounds good, added comment. thanks!